oidc-server/main.go

```diff
@@ -56,10 +60,54 @@ func NewOIDCServer(l *zap.Logger, c *OIDCServerConfig) (*OIDCServer, error) {
 		m.CodeChallengeMethodsSupported = c.PKCEChallengeMethodsSupported
 	}
 
+	// load user in the queue
+	if c.MockUserFolder != "" {
```

Comment:
You can do the dir walk directly here, no need for the ` GetMockedUsers` function
```golang
       // load user in the queue
	if c.MockUserFolder != "" {
		// go over each YAML file in mock user dir
		err := filepath.Walk(c.MockUserFolder, func(path string, info os.FileInfo, err error) error {
			if err != nil {
				l.Error("Error accessing file", zap.String("path", path), zap.Error(err))
				return err
			}

			// parse yaml files only
			if strings.HasSuffix(info.Name(), ".yml") || strings.HasSuffix(info.Name(), ".yaml") {
				user, err := ReadMockUser(path)
				if err != nil {
					log.Fatal(err)
				}

				l.Debug("adding mock user to queue", zap.String("user", user.ID()), zap.String("path", path))
				m.QueueUser(&user)
			}

			// skip other files
			return nil
		})
		if err != nil {
			l.Error("error walking dir", zap.String("dir", c.MockUserFolder), zap.Error(err))
			return nil, err
		}
```